solver/src/solver/http_solver.rs

```diff
@@ -449,28 +449,19 @@ impl HttpSolver {
 
 fn split_liquidity(
     liquidity: Vec<Liquidity>,
-) -> (
-    Vec<LimitOrder>,
-    Vec<ConstantProductOrder>,
-    Vec<WeightedProductOrder>,
-) {
-    let mut limit_orders = Vec::new();
+) -> (Vec<ConstantProductOrder>, Vec<WeightedProductOrder>) {
     let mut constant_product_orders = Vec::new();
     let mut weighted_product_orders = Vec::new();
     for order in liquidity {
         match order {
-            Liquidity::Limit(order) => limit_orders.push(order),
             Liquidity::ConstantProduct(order) => constant_product_orders.push(order),
             Liquidity::WeightedProduct(order) => weighted_product_orders.push(order),
         }
     }
-    (
-        limit_orders,
-        constant_product_orders,
-        weighted_product_orders,
-    )
+    (constant_product_orders, weighted_product_orders)
 }
 
+// TODO: This currently does **NOT** consider balancer pools, but definitely should.
 fn remove_orders_without_native_connection(
```

Comment:
That is weird... how is this software already running fine on weighted pools if they aren't here? Is this related to price estimation? Should we link this TODO in #1075?

Reply:
I already created a separate issue in https://github.com/gnosis/gp-v2-services/issues/1071

This was probably something that slipped by in the initial Balancer V2 weighted pool integration.